src/main/cpp/gamepad/TeleopControl.h

```diff
@@ -45,29 +45,42 @@ class TeleopControl
             DRIVE_75PERCENT,
             DRIVE_50PERCENT,
             DRIVE_25PERCENT,
-            DRIVE_SHIFT_UP,
-            DRIVE_SHIFT_DOWN,
-            DRIVE_TURBO,
-            DRIVE_BRAKE,
+            DRIVE_SHIFT_UP,  //not mapped to controller 2.13.22
+            DRIVE_SHIFT_DOWN,  //not mapped to controller 
+            DRIVE_TURBO,//not mapped to controller
+            DRIVE_BRAKE,//not mapped to controller
             REZERO_PIGEON,
-            ROTATE_TO_GOAL,
+            ROTATE_TO_GOAL,  //not used maybe same as FINDTARGET
             DRIVE_POLAR,
-            DRIVE_POLAR_INTAKE,
+            DRIVE_POLAR_INTAKE,   //not used yet?? 2.13.22
             INTAKE_LEFT,
             INTAKE_RIGHT,
             EXPEL_LEFT,
             EXPEL_RIGHT,
-            AUTO_SHOOT_HIGH,
-            AUTO_SHOOT_LOW,
-            CLIMBER_REACH,
-            CLIMBER_ROTATE,
-            AUTO_CLIMB_TRAVERSE,
-            LIMELIGHT_FEED_TO_DASH,
-            STOP_LIMELIGHT_FEED_TO_DASH,
-            CAMERA_FEED_TO_DASH,
-            STOP_CAMERA_FEED_TO_DASH,
-            
+            AUTO_SHOOT_HIGH,  //not used 
+            AUTO_SHOOT_LOW,   //not used
+            CLIMBER_REACH,   //not used
+            CLIMBER_ROTATE,  //not used
+            AUTO_CLIMB_TRAVERSE,  //not used
+            LIMELIGHT_FEED_TO_DASH, //not used
+            STOP_LIMELIGHT_FEED_TO_DASH,  //not used
+            CAMERA_FEED_TO_DASH,  //not used
+            STOP_CAMERA_FEED_TO_DASH,  //not used
+
+            // added 2022 Geo3 2.13.22
             FINDTARGET,    //Rotate to point to target center
+            CLIMBER_MAN_UP,  //Button Mapped needs calls at mgrs
+            CLIMBER_MAN_DOWN, //Button Mapped needs calls to mgrs
+            CLIMB_AUTO, //Button Mapped needs calls to mgrs
+            AUTO_SHOOT,//Button Mapped needs calls to mgrs
```

Comment:
Is this the same as AUTO_SHOOT_HIGH?   What about AUTO_SHOOT_LOW are we dropping that?